FWCore/Framework/src/EventProcessor.cc

```diff
@@ -486,13 +483,13 @@ namespace edm {
     ScheduleItems items;
 
     //initialize the services
-    std::shared_ptr<std::vector<ParameterSet> > pServiceSets = processDesc->getServicesPSets();
-    ServiceToken token = items.initServices(*pServiceSets, *parameterSet, iToken, iLegacy, true);
+    auto pServiceSets = processDesc->getServicesPSets();
```

Comment:
This is also forcing a copy. Is the copy needed? if not then it should be changed to `auto const&`.